reporter/reporter_test.go

```diff
@@ -41,18 +42,18 @@ import (
 )
 
 var (
-	compressor               = policy.NewAggregationIDCompressor()
-	compressedMax, _         = compressor.Compress(policy.AggregationTypes{policy.Max})
-	compressedP9999, _       = compressor.Compress(policy.AggregationTypes{policy.P9999})
-	compressedMaxAndP9999, _ = compressor.Compress(policy.AggregationTypes{policy.Max, policy.P9999})
+	compressor               = aggregation.NewIDCompressor()
+	compressedMax, _         = compressor.Compress(aggregation.Types{aggregation.Max})
```

Comment:
While you are here, can you change this and the following ones to use `MustCompress`?